validator/project_validator.go

```diff
@@ -1016,6 +1017,21 @@ func checkRunOn(runOnHasDistro, runOnHasContainer bool, runOn []string) []Valida
 	return nil
 }
 
+func validateTimeoutLimits(_ context.Context, settings *evergreen.Settings, project *model.Project, _ *model.ProjectRef, _ bool) ValidationErrors {
+	errs := ValidationErrors{}
+	if settings.TaskLimits.MaxTaskSecs > 0 {
```

Comment:
Exec timeout can be set dynamically during task runtime using the `timeout.update` command, so it also has to be validated when [dynamically setting it in the agent](https://github.com/evergreen-ci/evergreen/blob/74522833d8fbc68fb725685bbf6f42dbda2d6a50/agent/command/timeout.go#L69) as well.

Reply:
done (passed limit through task config because it was easiest to pass through)